sms_939/models/sms_notification.py

```diff
@@ -87,17 +86,19 @@ def run_service(self):
         sms_text = self.text or ''
         if not self.hook_id:
             hooks = self.env['sms.hook'].search([])
+            n1 = "\n"
             sms_answer = _(
                 "Sorry, we could not understand your request. "
-                "Supported services are :\n - %s "
-            ) % "\n- ".join(hooks.mapped('name'))
+                f"Supported services are :\n - {n1 + '- '.join(hooks.mapped('name'))}"
```

Comment:
Why putting \n in a variable?
```suggestion
                f"Supported services are :\n - {'\n-  '.join(hooks.mapped('name'))}"
```

Reply:
Because expression fragments inside f-strings cannot includes backslashes